README.md

```diff
@@ -38,17 +39,32 @@ To setup the project locally read these wiki pages and follow the instructions:
 
 The project runs on Python 3.
 
-1. Create a virtual environment:
-`virtualenv venv --python=python3`
+1. Install latest version of virtualenv:
+```
+pip install -U virtualenv
+```
 
-2. Activate the virtual environment:
-`source /venv/Scripts/activate` for Git Bash Users, 
-`venv\Scripts\activate` for Windows Command Line Users
+2. Create a virtual environment:
+```
+virtualenv venv --python=python3
+```
 
-3. Install all the dependencies in `requirements.txt` file:
-`pip install -r requirements.txt`
+3. Activate the virtual environment:
+```
+source /venv/Scripts/activate
+```
+for Git Bash Users,
```

Comment:
If possible do something with this line. It looks odd to me.

Reply:
DO something, do what?

Reply:
![image](https://user-images.githubusercontent.com/42068010/93736508-35bc0180-fbfe-11ea-9d69-befc6d8f86d3.png)
I think it's better if we reverse the flow of command and description shown and use capitals and colon symbol.

Reply:
Done